### PR TITLE
Added help text to EmailInput component

### LINKED ISSUE #845: Add helpText for Email Input
Please add helpText props to Email Input

cc @karthiknmenon @amaldinesh7

### PR DESCRIPTION
**Pull request by Thejus-Paul**
Fixes #845 
- Added help text to EmailInput component and its story.
- Added default props for `error` and `label`.
- Rearranged the `index.js` file in alphabetic order.
- Added feature to prune similar values.
- Added a custom yup validation test.

### CODE REVIEW

**Comment by Thejus-Paul** [comment]
help_text_email_component

@karthiknmenon _a Please review.

**Comment by Thejus-Paul** [comment]
prune_duplicate_values
`react-select`'s multi-select text input field has some issues while removing duplicate values. The delete operation will not work as shown above. Since EmailInput component was built using this, it has inherited this issue. Thus, I have added a simple fix to prune the duplicate values.

**Comment by karthiknmenon** [comment]
@ajmaln _a Please review.

File: lib/components/EmailInput.js [hunk 1]
```diff
@@ -109,14 +112,18 @@ EmailInput.propTypes = {
...
   /**
    * The values to be displayed inside the input field.
    */
   value: PropTypes.array,
   /**
-   * To specify the error message to be shown in the input field.
+   * To specify the error message to be shown below the input field.
    */
-  error: PropTypes.string,
+  error: PropTypes.oneOfType([String, Array]),
```
Comment (amaldinesh7) [inline lib/components/EmailInput.js:126]: Let's keep only the string type for the error prop. Please make the changes as discussed over the call. Let me know if there is any difficulty with that.

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@Thejus-Paul _a Please check the comments.

**Comment by Thejus-Paul** [comment]
@amaldinesh7 _a Please review.

**Review by amaldinesh7 — APPROVED** [review]

**Pull request merged**